Tests/SwiftProtobufTests/Test_AllTypes_Proto3.swift

```diff
@@ -777,13 +777,17 @@ class Test_AllTypes_Proto3: XCTestCase, PBTestHelpers {
         assertDecodeFails([253, 1, 77])
         assertDecodeFails([254, 1, 78])
         assertDecodeFails([255, 1, 79])
-        assertDebugDescription("Proto3TestAllTypes(repeatedInt32:[1])") {(o: inout MessageTestType) in
+        // WHY ISN'T THIS LIKE PROTO2?
+        //   "Proto3TestAllTypes:\nrepeated_int32: 1\n"
+        assertDebugDescription("Proto3TestAllTypes:\nrepeated_int32: [1]\n") {(o: inout MessageTestType) in
```

Comment:
Ok, land this as is and do the tweaks you mentioned in your cl (and fix up these tests to match the changes)?